Libs/Optimize/ParticleSystem/itkParticleImageDomain.h

```diff
@@ -11,6 +11,10 @@
 #include "itkImage.h"
 #include "itkParticleRegionDomain.h"
 #include "itkLinearInterpolateImageFunction.h"
+#include <itkZeroCrossingImageFilter.h>
+#include <vtkContourFilter.h>
+#include <vtkMassProperties.h>
+#include <debug-install/include/ITK-5.0/itkImageRegionConstIteratorWithIndex.h>
```

Comment:
This include is weird. I replaced it with #include <itkImageRegionConstIteratorWithIndex.h>

Reply:
Oops, thanks.